Source/Core/Core/HW/WiimoteEmu/MotionPlus.cpp

```diff
@@ -4,10 +4,54 @@
 
 #include "Core/HW/WiimoteEmu/MotionPlus.h"
 
+#include <algorithm>
+#include <cmath>
+
+#include <mbedtls/bignum.h>
+#include <zlib.h>
+
 #include "Common/BitUtils.h"
 #include "Common/ChunkFile.h"
 #include "Common/Logging/Log.h"
+#include "Common/MathUtil.h"
 #include "Common/MsgHandler.h"
+#include "Common/Swap.h"
+
+#include "Core/HW/Wiimote.h"
+#include "Core/HW/WiimoteEmu/Dynamics.h"
+
+namespace
+{
+// Minimal wrapper mainly to handle init/free
+struct MPI : mbedtls_mpi
```

Comment:
Note to self: "Multi-Precision Integer". Only knew those things as "Big Integer" before, good to know.

Reply:
:+1: